as2/6.0/modules/ROOT/pages/as2-connector-upgrade-migrate.adoc

```diff
@@ -1,6 +1,8 @@
-= AS2 Connector 5.0.0 - Upgrade and Migrate - Mule 4
+= Upgrade AS2 - Mule 4
```

Comment:
```suggestion
= Upgrade AS2 Connector - Mule 4
```

Reply:
Not sure if you can enter 6.0. Per previous edits in this doc the number 6.0.0 was changed to 6.x